src/display.js

```diff
@@ -162,12 +162,13 @@ const Display = {
             const listItemElement = listElement.querySelector('#listPoke' + index);
             if (listItemElement) {
                 const listItemNameElement = listItemElement.querySelector('.pokeListName');
-                let hasChanged = (listItemNameElement.innerHTML !== `${poke.pokeName()} (${poke.level()})`) || (listItemNameElement.getAttribute('status') !== this.pokeStatus(poke));
-                listItemNameElement.innerHTML = `${poke.pokeName()} (${poke.level()})`;
+                let hasChanged = (listItemNameElement.innerHTML !== `${poke.pokeName()} (${poke.level() + (poke.prestigeLevel ? ("p" + poke.prestigeLevel) : "")})`) || (listItemNameElement.getAttribute('status') !== this.pokeStatus(poke));
```

Comment:
This should probably get its own function as well, in case we want to change the format